bindings/wasm/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+      "declaration": true,
+      "outDir": "dist",
```

Comment:
If we're using `ts-node` there should be no `outDir` unless I'm missing something. It's fine anyways.